### PR TITLE
bdd-test-list-suppliers working on the BDD list supplier test

### PR DESCRIPTION
**Pull request by sa3613**
We are still working on BDD list supplier test, Bea to investigate

### CODE REVIEW

**Comment by evpoe** [comment]
This PR is ready for review/merging! See commits for details

File: features/suppliers.feature [hunk 1]
```diff
@@ -38,20 +38,23 @@ Scenario: Create a Supplier
...
-#     And I should not see "leo" in the results
+Scenario: Search a Supplier
+    When I visit the "Home Page"
+    And I press the "Clear" button
+    And I set the "name" to "Evan"
+    And I press the "Search" button
+    Then I should see "Evan" in the "name" field
+    Then I should see "evan@titans.com" in the "email" field
+    Then I should see "14 Cashville Ln" in the "address" field
+    Then I should see "9991235575" in the "phone_number" field
```
Comment (CatherineRH) [inline features/suppliers.feature:57]: @sa3613 @evpoe should this also have a "Then I should NOT see..."?

Comment (evpoe) [reply features/suppliers.feature:57]: @CatherineRH good question but I'm having trouble figuring out what we would NOT want to see in this scenario...what're you thinking?

Comment (CatherineRH) [reply features/suppliers.feature:57]: We should not see Sophie, Catherine, Sam, or Bea right?

Comment (sa3613) [reply features/suppliers.feature:57]: @CatherineRH this is a good point, is it necessary to add? If yes I can add them. Let me know!

Comment (evpoe) [reply features/suppliers.feature:57]: it's a valid point but if we want to go that route I would recommend a different set of code...something like

`  Then I should see "Evan" in the results
    And I should not see "Sophie" in the results
    And I should not see "Catherine" in the results
    And I should not see "Sam" in the results
    And I should not see "Bea" in the results`

Comment (evpoe) [reply features/suppliers.feature:57]: the current test looks to see if the "fields" were populated correctly (a little redundant with our CREATE bdd test....while this updated version does a better job ensuring the "results" were populated correctly

two different tests but the version catherine is talking about might be more what we're going after!

Comment (CatherineRH) [reply features/suppliers.feature:57]: I'm fine with either also, but yes, Evan that's exactly what I was thinking

Comment (evpoe) [reply features/suppliers.feature:57]: On it

**Pull request merged**